File: solutions/beecrowd/1962/1962.cpp
#include <cstdint>
#include <iostream>

int main() {
    int32_t n, t, year;

    std::cin >> n;

    while (n--) {
        std::cin >> t;
        year = 2015 - t;

        if (year <= 0) {
            year--;
        }

        std::cout << std::abs(year) << (year < 0 ? " A.C." : " D.C.")
                  << std::endl;
    }

    return 0;
}

File: solutions/beecrowd/1962/1962.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

n = int(input())

for _ in range(n):
    t = int(input())
    year = 2015 - t

    if year <= 0:
        year -= 1

    output(f'{abs(year)} {"A.C." if year < 0 else "D.C."}\n')
